src/main/java/com/ahoo/issuetrackerserver/member/Member.java

```diff
@@ -56,4 +56,8 @@ public static Member of(String loginId, String password, String email, String ni
         String profileImage, AuthProvider authProviderType, String resourceOwnerId) {
         return new Member(null, loginId, password, email, nickname, profileImage, authProviderType, resourceOwnerId);
     }
+
+    public boolean isCorrectPassword(String password) {
+        return this.password.toLowerCase().equals(password);
```

Comment:
```suggestion
        return this.password.equalsIgnoreCase(password);
```

만약, 대소문자 구분 안하기로 한거라면 이렇게 하는게 좋을 거 같아요. 호출하는 쪽에서도 LowerCase를 임의로 해준 값을 넣어줘야 하는 구조라서 모르는 사람이 쓰면 휴먼에러가 발생할 가능성이 높아보입니다.

Reply:
`.equalsIgnoreCase()`를 깜빡했네요 😅